packages/french-determiners/src/index.ts

```diff
@@ -23,6 +23,20 @@ type getDetParameters = {
   forceDes?: boolean;
 };
 
+/* 
+  ma, ta, sa deviennent mon, ton, son devant des noms féminins commençant par une voyelle pour éviter le hiatus de deux voyelles. 
+  a priori, ce n'est pas le cas avec y: par exemple ma yogourtière
+  https://open.byu.edu/grammaire_ouverte/determinants#:~:text=N'oubliez%20pas%20de%20faire,le%20hiatus%20de%20deux%20voyelles.
+*/
+function getFixHiatus(contentAfterDet: string) {
+  if (!contentAfterDet) {
+    return false;
+  }
+  const firstLetterAfter = contentAfterDet.replace(/¤/g, ' ').trim().substring(0, 1);
+  const fixHiatus = ['a', 'e', 'é', 'è', 'ê', 'i', 'o', 'u'].includes(firstLetterAfter);

```

Comment:
Can you use the full list? àáâãäåÀÁÂèéêëÈÉÊËìíîïÌÍÎÏòóôõöøÒÓÔÕÖØùúûüÙÚÛÜ

(we will barely find a word starting with ø in French, but capital letters can happen)

Reply:
I also add support for the h muet